lib/route.js

```diff
@@ -312,7 +315,15 @@ function buildRouting (options) {
       }
     }
 
-    var childLogger = logger.child({ [requestIdLogLabel]: req.id, level: context.logLevel })
+    var loggerOpts = {
+      [requestIdLogLabel]: req.id,
+      level: context.logLevel
+    }
+
+    if (context.logSerializers) {
```

Comment:
This is in conflict with the default value because it will be always truty, for this reason, I think `null` is better

Reply:
I agree with you! Updated!